packages/eslint-plugin/src/lib/util.ts

```diff
@@ -0,0 +1,144 @@
+import { Rule } from 'eslint';
+import { ParserServices } from '@typescript-eslint/typescript-estree';
+
+const version = require('../../package.json').version;
```

Comment:
we should try use `import packageJson from '../../package.json';` then set `const version = packageJson.version` (this works if tsc has `resolveJsonModule: true`)

Reply:
I migrated the index to use a .json import, but I missed this one!
However, I moved this, but changing it to an import makes TSC copy the package.json to the dist folder, which breaks the build.

![image](https://user-images.githubusercontent.com/7462525/51565499-03fe0080-1e47-11e9-94ac-dc08f536aa1c.png)

Reply:
yup that's a shame, i hadn't realised the limitation 😞  guess we have to stick with a require